src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Services/SyntaxFacts/CSharpSyntaxFacts.cs

```diff
@@ -1726,6 +1733,9 @@ public SyntaxNode GetExpressionOfAwaitExpression(SyntaxNode node)
         public SyntaxNode GetExpressionOfThrowExpression(SyntaxNode node)
             => ((ThrowExpressionSyntax)node).Expression;
 
+        public SyntaxNode GetStatementOfGlobalStatement(SyntaxNode node)
+            => ((GlobalStatementSyntax)node).Statement;
```

Comment:
what is this for?

Reply:
removed, it was when I was special casing for global statements. That code got removed.